Makefile

```diff
@@ -156,13 +167,12 @@ flake8:
 		--copyright-check \
 		--copyright-min-file-size=1 \
 		--copyright-regexp="Copyright \(c\) ([0-9][0-9][0-9][0-9]-)?$(shell date +%Y), Camptocamp SA"
-	flake8 --max-line-length=110 \
+	git grep --files-with-match '/usr/bin/env python' | grep -v Makefile | xargs flake8 --max-line-length=110 \
```

Comment:
Is it not possible to simple use config file for flake 8 (setup.cfg)

Reply:
I prefer to have all the part of the project that use the same way...

Reply:
Done but it's not a big simplification...